seahorn/jobs2/priority_queue_capacity2/aws_priority_queue_capacity_harness2.c

```diff
@@ -12,38 +12,29 @@ int main(void) {
   initialize_priority_queue(&queue);
 
   /* assumptions */
-  #ifdef __KLEE__
-    // Inside aws_priority_queue_is_valid, the priority queue require the following be True
-    // Use the following code to exclude unqualified program path
-    if (!(queue.backpointers.item_size == sizeof(struct aws_priority_queue_node *)))
-      return 0;
-  #endif
   assume(aws_priority_queue_is_bounded(&queue, MAX_INITIAL_ITEM_ALLOCATION,
                                        MAX_ITEM_SIZE));
   assume(aws_priority_queue_is_valid(&queue));
   assume(queue.container.item_size > 0);
 
   /* save current state of the container structure */
-  struct aws_array_list old_container = queue.container;
-  struct store_byte_from_buffer old_byte_container;
-  save_byte_from_array((uint8_t *)old_container.data,
-                       old_container.current_size, &old_byte_container);
-
-  /* save current state of the backpointers structure */
-  struct aws_array_list old_backpointers = queue.backpointers;
-  struct store_byte_from_buffer old_byte_backpointers;
-  save_byte_from_array((uint8_t *)old_backpointers.data,
-                       old_backpointers.current_size, &old_byte_backpointers);
+  sea_tracking_on();
+  sea_reset_modified((char *)&queue.container);
```

Comment:
If my understanding is correct, also based on example from byte_buf, this should be correct?

Reply:
I am not sure. Why prefer this over comparing `queue.container` field by field.